src/toniarts/openkeeper/game/controller/MapController.java

```diff
@@ -285,17 +285,17 @@ public void selectTiles(Vector2f start, Vector2f end, boolean select, short play
     }
 
     @Override
-    public boolean isSelected(int x, int y, short playerId) {
-        MapTile tile = getMapData().getTile(x, y);
+    public boolean isSelected(Point p, short playerId) {

```

Comment:
Overload would have been better. Now we always need to construct a point. I guess the performance and memory hit is quite marginal, even I at some point wanted to have a point cache. Waiting for tuples to come to Java as well. Then we wouldn't need to construct these objects.

Reply:
We must use the Point everywhere in methods. It is better to load from files to a Point and getX() methods, getY() to do through a Point.

Reply:
Ok